index.html

```diff
@@ -20,15 +20,59 @@
         <script type="module">
             import { header } from './header.js';
             import { consoleTempl } from './htmlConsole.js';
-
+            import { members } from './members.js';
+            
             const template = `
                 <div>
                     ${ header }
                     ${ consoleTempl }
+                    ${ members }
                 </div>
             `;
-
+            
             document.getElementById("main").innerHTML = template;
         </script>
+
+                <script type="text/javascript" language="javascript">
+                    var http_request = false;
+                    var members = null;
+
+                    function loadJSON(callback) {   
+                        var xobj = new XMLHttpRequest();
+                            xobj.overrideMimeType("application/json");
+                        xobj.open('GET', 'members.json', true); // Reads content from members.json
+                        
+                        xobj.onreadystatechange = function () {
+                            if (xobj.readyState == 4 && xobj.status == "200") {
+                                callback(xobj.responseText);
+                                members.forEach(element => {
+                                    let member = document.createElement("li");
+                                    let css_id = (element.leader) ? "leader" : "member";
+                                    member.classList.add(css_id + '-flex-item');
+                                    member.innerHTML = `
+                                            <div class="`+css_id+`-image-cropper">
```

Comment:
Para esto podrías usar [`Template literals`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) sin ningún problema . Sería algo como : 
```suggestion
                                            <div class="${css_id}-image-cropper">
```